app/views/docs/rest.phtml

```diff
@@ -1,12 +1,310 @@
+
+<p>The multipart form data posted to file upload endpoints has the following format:</p>
+
+<div class="ide margin-bottom" data-lang="http" data-lang-label="HTTP">
+    <pre class="line-numbers"><code class="prism language-http" data-prism>POST /v1/storage/buckets/default/files  HTTP/1.1
+Host: [HOSTNAME_OR_IP]
+Content-Type: multipart/form-data; boundary=---WD9146A
+X-Appwrite-Project: demo-project
+content-range: bytes 10485760-12582912/12582912
+x-appwrite-id: 6369b0bc1dcf4ff59051
+
+------WebKitFormBoundarye0m6iNBQNHlzTpVM
+Content-Disposition: form-data; name="fileId"
+
+unique()
+------WebKitFormBoundarye0m6iNBQNHlzTpVM
+Content-Disposition: form-data; name="file"; filename="file.txt"
+Content-Type: application/octet-stream
+
+[CHUNKED-DATA]
+------WebKitFormBoundarye0m6iNBQNHlzTpVM
+Content-Disposition: form-data; name="permissions"
+
+["read(\"user:627a958ded6424a98a9f\")"]
+------WebKitFormBoundarye0m6iNBQNHlzTpVM--</code></pre>
+</div>
+
+<h2>Permissions</h2>
+
+<p>Appwrite SDKs have helpers to generate permission strings, but when using Appwrite without SDKs, you'd need to create the strings yourself.</p>
+
+<h3>Permission Types</h3>
+
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 250px">SDK</td>
+        <td>Permission String</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><code>Permission.read()	</code></td>
+        <td><code>"read(\"[PERMISSION_ROLE]\")"</code></td>
+    </tr>
+    <tr>
+        <td><code>Permission.create()</code></td>
+        <td><code>"create(\"[PERMISSION_ROLE]\")"</code></td>
+    </tr>
+    <tr>
+        <td><code>Permission.update()</code></td>
+        <td><code>"update(\"[PERMISSION_ROLE]\")"</code></td>
+    </tr>
+    <tr>
+        <td><code>Permission.delete()</td>
+        <td><code>"delete(\"[PERMISSION_ROLE]\")"</code></td>
+    </tr>
+    <tr>
+        <td><code>Permission.write()</td>
+        <td><code>"write(\"[PERMISSION_ROLE]\")"</code></td>
+    </tr>
+    </tbody>
+</table>
+
+<h3>Permission Roles</h3>
+
+<table class="full text-size-small">
+    <thead>
+    <tr>
+        <td style="width: 250px">SDK</td>
+        <td>Role String</td>
+    </tr>
+    </thead>
+    <tbody>
+    <tr>
+        <td><code>Role.any()</code></td>
+        <td><code>"any"</code></td>
+    </tr>
+    <tr>
+        <td><code>Role.guests()</code></td>
+        <td><code>"user:[ID]"</code></td>
+    </tr>
+    <tr>
+        <td><code>Role.users([STATUS])</code></td>
+        <td><code>"users/[STATUS]"</code></td>
+    </tr>
+    <tr>
+        <td><code>Role.user([USER_ID], [STATUS])</td>
+        <td><code>"user:[USER_ID]/[STATUS]"</code></td>
+    </tr>
+    <tr>
+        <td><code>Role.team([TEAM_ID])</td>
+        <td><code>"team:[TEAM_ID]"</code></td>
+    </tr>
+    <tr>
+        <td><code>Role.team([TEAM_ID], [ROLE])</td>
+        <td><code>"team:[TEAM_ID]/[ROLE]"</code></td>
+    </tr>
+    </tbody>
```

Comment:
`Role.member()` should be included too, right? See [sdk-for-web](https://github.com/appwrite/sdk-for-web/blob/master/src/role.ts#L31-L32).

Reply:
Good call. We might be missing this in the DB docs actually 😰

Reply:
FYI, you're supposed to pass a membership ID to it so it grants access to a user while they're still part of the team.